### PR TITLE
docs: Add router example docs to networking section

### PR DESCRIPTION
**Pull request by jsilverio22**
Issue: #749 
Signed-off-by: Joshua Silverio <joshua@acorn.io>

### CODE REVIEW

File: docs/docs/50-running/02-networking.md [hunk 1]
```diff
@@ -120,3 +120,49 @@ To highlight the level of control this gives you, consider the following:
 If you set your cluster domain to `my-company.com`, deployed into the default `acorn` namespace, named your app `blog`, and the container name in the Acornfile was `default`, the published endpoint for your app would be: `http://blog.my-company.com`
 
 In addition to this method of controlling your endpoint, you can publish to an explicit external name using the `--publish` flag. See the [publishing ports](#publish-individual-ports) section for more details.
+## Routers
```
Comment (tylerslaton) [inline docs/docs/50-running/02-networking.md:123]: Just a nit-pick, but could we add newlines to match the rest of this document?
```suggestion

## Routers

```

File: docs/docs/50-running/02-networking.md [hunk 2]
```diff
@@ -120,3 +120,49 @@ To highlight the level of control this gives you, consider the following:
...
+If you have two containers normally they would be exposed as two different HTTP services. The router feature allows you to expose these two containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this below.
```
Comment (tylerslaton) [inline docs/docs/50-running/02-networking.md:124]: What are your thoughts on this? Personally I think that it makes it clearer that this scales to more than two containers which may be non-obvious for beginners.

```suggestion
If you have multiple containers normally they would be exposed as two different HTTP services. The router feature allows you to expose those containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this with two containers below.
```

File: docs/docs/50-running/02-networking.md [hunk 3]
```diff
@@ -120,3 +120,49 @@ To highlight the level of control this gives you, consider the following:
...
+
+Starting with a sample Acornfile that exposes two services
+```acorn
```
Comment (tylerslaton) [inline docs/docs/50-running/02-networking.md:127]: Follow up from the previous nit-pick, could we try to make the formatting with newlines match the rest of the document? Applies to the rest of this document.

```suggestion

```acorn

```

**Review by tylerslaton — COMMENTED** [review]
Nice! Just had a couple of small comments.

File: docs/docs/50-running/02-networking.md (hunk 2 above)
Comment (jsilverio22) [reply docs/docs/50-running/02-networking.md:124]: I like this better

File: docs/docs/50-running/02-networking.md (hunk 3 above)
Comment (jsilverio22) [reply docs/docs/50-running/02-networking.md:129]: fixed

**Review by tylerslaton — APPROVED** [review]
Looks good to me 🚀

**Review by cjellick — APPROVED** [review]

**Pull request merged**